vue/src/types/widgetOption.ts

```diff
@@ -0,0 +1,45 @@
+export interface WidgetOption {
+  label: string;
+  name: string;
+}
+
+export const widgetOptions = {
+  weather: {
+    label: "weather",
+    name: "weather",
+  },
+  soilParameter: {
+    label: "soil parameter",
+    name: "soil-parameter",
+  },
+  notification: {
+    label: "notifications",
+    name: "notifications",
+  },
+  gardenMap: {
+    label: "garden map",
+    name: "garden-map",
+  },
+  cropsTable: {
+    label: "crops table",
+    name: "crops-table",
+  },
+  cropsMap: {
+    label: "crops map",
+    name: "crops-map",
+  },
+  "3dTable": {
+    label: "3D table",
+    name: "3d-table",
+  },
+  "3dMap": {
+    label: "3D map",
+    name: "3d-map",
```

Comment:
Here perhaps just a name without a number at the beginning?

- `3dTable` -> `crops3dTable` ? 
- `3dMap` -> `crops3dMap` ?

Reply:
good idea 👍